docs/content/users/configuration/config.md

```diff
@@ -353,6 +330,19 @@ When `true`, the `config.*.yaml` file with the option will have its settings *ov
 
 See [Extending `config.yaml` with Custom `config.*.yaml` Files](../extend/customization-extendibility.md#extending-configyaml-with-custom-configyaml-files).
 
+## `performance_mode`
```

Comment:
Somewhere somehow we need to add --performance-mode-reset.  I think the details of this can be added to `performance_mode` since there's not an equivalent global setting.

Reply:
It's mentioned in the commands docs see https://github.com/ddev/ddev/pull/4904/files#diff-b8208bb33ec0489e78b5a8fdc3d7a5623dbcd64851d2a4633ce651bcce2c9853. Do you think we need another hint here?

Reply:
Let's go with it as it is and see how the future holds.